src/constants.js

```diff
@@ -0,0 +1,2 @@
+const SOURCE_SDK = "javascript";
+const SOURCE_SDK_VERSION = "2.0.0";
```

Comment:
Rather than assigning this as a constant, we should get this from package.json so they don't get out of sync.

Reply:
That would be great!